dashboard-api/internal/platform/server/server.go

```diff
@@ -55,13 +56,18 @@ func (s *Server) registerRoutes() {
 
 	s.Engine.Use(cors.New(config))
 
+	// API for Health Check
 	s.Engine.GET("/health", health.CheckHandler(s.repositories, s.auth0Domain))
 
+	//APIS used for the studies resource
 	s.Engine.GET("/studies/:slug", s.ensureValidTokenMiddleware, studies.ReadHandler(s.repositories))
 	s.Engine.GET("/studies", s.ensureValidTokenMiddleware, studies.ListHandler(s.repositories))
 	s.Engine.POST("/studies", s.ensureValidTokenMiddleware, studies.CreateHandler(s.repositories))
-
 	s.Engine.GET("/studies/:slug/segments-progress", s.ensureValidTokenMiddleware, segmentsprogress.ListHandler(s.repositories))
 
+	//APIS for the study configurations resource
+	s.Engine.POST("/studies/:slug/configuration", s.ensureValidTokenMiddleware, studyconfig.CreateHandler(s.repositories))
```

Comment:
While I appreciate the specificity, given that the types and tables everywhere are called "confs", it seems almost confusing to switch to a more proper word.

Reply:
Will change it to StudyConf